packages/editor/src/components/parts/task/options/TaskListOptions.tsx

```diff
@@ -10,6 +10,7 @@ const TaskListPart = () => {
   return (
     <PathCollapsible label='Options' defaultOpen={task.skipTasklist} path='delay'>
       <Checkbox label='Skip Tasklist' value={task.skipTasklist} onChange={change => update('skipTasklist', change)} />
+      <Checkbox label='Suppress Notification' value={task.notification.suppress} onChange={change => update('notification', {suppress: change})} />
```

Comment:
just as info: this works as long as you don't have another property in the notification object
But ok for me